pkg/services/sqlstore/migrations/ualert/channel.go

```diff
@@ -220,32 +238,40 @@ func makeKeyForChannelGroup(channelUids map[interface{}]struct{}) (string, error
 }
 
 // addDefaultChannels should be called before adding any other routes.
-func (m *migration) addDefaultChannels(amConfig *PostableUserConfig, allChannels map[interface{}]*notificationChannel, defaultChannels []*notificationChannel) error {
-	// Default route and receiver.
-	recv, route, err := m.makeReceiverAndRoute("default_route", nil, defaultChannels, allChannels)
-	if err != nil {
-		return err
-	}
+func (m *migration) addDefaultChannels(amConfigsPerOrg amConfigsPerOrg, allChannels channelsPerOrg, defaultChannels defaultChannelsPerOrg) error {
+	for orgID := range allChannels {
```

Comment:
If there is some org without any notification channels yet, then no default route will be added to that org here. But I think we need at least 1 default routing policy (and a contact point to attach to that) per org. If we have a list of all orgs, then it would be easiest to iterate through that list here instead of `allChannels` (which will end up adding an empty contact point and default routing policy for such orgs).

Reply:
How the result configuration should look like in case there are not any notification channels associated with an organisation?
Does the following makes sense?
```
{
	"template_files": null,
	"alertmanager_config": {
		"route": {
			"receiver": "autogen-contact-point-default"
		},
		"templates": null,
		"receivers": [
			{
				"name": "autogen-contact-point-default"
			}
		]
	}
}
```

Reply:
You are right, it is best to store nothing in that case and go with the default config that we store. Please ignore this comment :)